src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -57,9 +62,23 @@ def _check_model_args(
     """
     if not args.subcommand and not args.model:
         parser.error("The -m/--model option is required and cannot be empty.")
+    args = _convert_str_to_enum_entry(
+        args, "model_selection_strategy", ModelSelectionStrategy
+    )
+    _generate_formatted_model_name(args)
     return args
 
 
+def _generate_formatted_model_name(args: argparse.Namespace) -> None:
+    if len(args.model) == 1:
+        args.formatted_model_name = args.model[0]
```

Comment:
Is this the right variable name? It feels a bit misleading to me.

It's the filename, not the model name, right?

Reply:
its also used as the model name passed to PA on the cli.

Reply:
If there are multiple models here, won't this break PA since this is adding _multi? Wouldn't PA act the same if it just received the first model name?

Reply:
It does not. We actually had a conversation about it and thankfully the model name is pulled from the prompt input.
I had the same concern

Reply:
Why does PA need this then?

Reply:
Elias explained that while the client uses the prompt input for the model name, it still needs any arbitrary model name, so this will be used.